app/src/main/java/com/ucmobiledevelopment/freeways/ReportIncidentActivity.kt

```diff
@@ -18,21 +17,22 @@ class ReportIncidentActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContent {
-            FreeWaysTheme() {
-                // A surface container using the 'background' color from the theme
+            FreeWaysTheme {
+                //A surface container using the 'background' color from the theme
                 Surface(
                     modifier = Modifier.fillMaxSize(),
                     color = MaterialTheme.colors.background
                 ) {
-                  //  Greeting("Android")
+
                 }
                 val context = LocalContext.current
-                ExtendedFloatingActionButton (
-                    text = {  Text(text = "Back") },
-                    onClick = { val intent = Intent(context, MainActivity::class.java)
-                                context.startActivity(intent)
-                              },
-                    icon = { Icon(Icons.Filled.ArrowBack,"@drawable/back_icon") },
+                ExtendedFloatingActionButton(
+                    text = { Text(text = "Back") },
+                    onClick = {
+                        val intent = Intent(context, MainActivity::class.java)
+                        context.startActivity(intent)
+                    },
+                    icon = { Icon(Icons.Filled.ArrowBack, "@drawable/back_icon") },
                     backgroundColor = Purple500
                 )
             }
```

Comment:
Reformated the code. 4 spaces and with proper indentation.  https://kotlinlang.org/docs/coding-conventions.html#formatting